packages/legend-application-studio/src/components/editor/edit-panel/project-configuration-editor/ProjectConfigurationEditor.tsx

```diff
@@ -360,13 +438,106 @@ export const ProjectConfigurationEditor = observer(() => {
   };
   const disableAddButton =
     selectedTab !== CONFIGURATION_EDITOR_TAB.PROJECT_DEPENDENCIES || isReadOnly;
-  const updateConfigs = (): void => {
-    editorStore.localChangesState.alertUnsavedChanges((): void => {
-      flowResult(configState.updateConfigs()).catch(
-        applicationStore.alertUnhandledError,
+
+  const updateGavConfigs = (): void => {
+    const documentationEntry =
+      applicationStore.documentationService.getDocEntry(
+        LEGEND_STUDIO_DOCUMENTATION_KEY.QUESTION_HOW_TO_UPDATE_PROJECT_GAV_COORDINATES,
       );
+
+    const actions = documentationEntry
+      ? [
+          {
+            label: 'Acknowledge and Proceed',
+            type: ActionAlertActionType.PROCEED_WITH_CAUTION,
+            handler: (): void => {
+              editorStore.localChangesState.alertUnsavedChanges((): void => {
+                flowResult(configState.updateConfigs()).catch(
+                  applicationStore.alertUnhandledError,
+                );
+              });
+            },
+          },
+          {
+            label: '',
+            content: (
+              <>
+                View Instructions
+                <div className="icon--right">
+                  <ExternalLinkIcon />
+                </div>
+              </>
+            ),
+            default: true,
+            type: ActionAlertActionType.PROCEED,
+            handler: (): void => {
+              if (
+                shouldDisplayVirtualAssistantDocumentationEntry(
+                  documentationEntry,
+                )
+              ) {
+                applicationStore.assistantService.openDocumentationEntry(
+                  'grammar.class',
+                );
+              } else if (documentationEntry.url) {
+                applicationStore.navigator.visitAddress(documentationEntry.url);
+              }
+            },
+          },
+          {
+            label: 'Cancel',
+            type: ActionAlertActionType.PROCEED,
+          },
+        ]
+      : [
+          {
+            label: 'Acknowledge and Proceed',
+            type: ActionAlertActionType.PROCEED_WITH_CAUTION,
+            handler: (): void => {
+              editorStore.localChangesState.alertUnsavedChanges((): void => {
+                flowResult(configState.updateConfigs()).catch(
+                  applicationStore.alertUnhandledError,
+                );
+              });
+            },
+          },
+          {
+            label: 'Cancel',
+            type: ActionAlertActionType.PROCEED,
+            default: true,
+          },
+        ];
+
+    editorStore.applicationStore.setActionAlertInfo({
+      message:
+        'Please read these instructions before you edit groupid or artifactid (gav) as changing them can have downstream impact. Be aware you will lose any previous project versions. Moreover, if your current project has dependant projects you can break those too if you do not change the gav in a controlled way.',
```

Comment:
```suggestion
        'Please be cautious that modifying group ID or artifact ID (GAV coordinates) can potentially have a big downstream impact. Be aware that the project will lose all previous versions; also, any dependant projects can break too if the coordinates are not changed in a controlled way',
```

Reply:
we should also add a prompt here: if `Boolean(documentationEntry)` we can show: `Please see the instructions for more guidance`